### ISSUE TITLE
Plugin asks user where to generate files to

### ISSUE THREAD

**Issue by NottyCode** [eclipse/codewind-openapi-vscode#47]
I have a maven project in vs code and the tool asks me where to write the generated Java source files. Given this is a maven project the tool should know to write the generated code into the `src/main/java` folder. Perhaps `src/test/java` might be a contender, but even then the question should be should this go into main or test, not which folder under src should it go into.

**Comment by NottyCode** [comment]
I raised this while waiting for the generation to complete. I had selected `src/main/java` as the folder not realizing that I should have provided the parent of `src`. So I now have `src/main/java/src/main/java`

**Comment by keithchong** [comment]
Hi @NottyCode , thanks for reporting the issues.  I am aware of some of these issue where there needs to be tweaks in the generator and wizard to apply to the various project types and languages.   See https://github.com/eclipse/codewind/issues/79

Another example is for Swift where the source would be located in the Sources folder.  I have code in place to default to the Sources folder, but I did not enable it.

Currently, the extension is 'general' enough to apply to all the languages that the OpenAPI generator supports, including the main languages that Codewind supports.   This means, unfortunately, there might be some level of refactoring involved by the user after the code has been generated.

**Comment by keithchong** [comment]
One more point, we documented this in the trouble-shooting section of the docs.   See the last paragraph here:

https://www.eclipse.org/codewind/troubleshooting.html#openapi-tools

**Comment by NottyCode** [comment]
I’m sorry but I don’t think expecting an end user to refactor generates code providers a good enough user experience. The fact that the code tries to cope with multiple languages doesn’t excuse this. That is essentially a code wind problem not a user problem. 

The trouble shooting docs aren’t available from vscode so how do you expect me to find it? In either case there are fundamental usability issues with these tools which would drive most people away from using them. Tools should be usable without documentation and these just aren’t.

**Comment by keithchong** [comment]
I agree with you 100%.  There is a plan in place to address this issue and it is on our radar.

WRT finding help, yeah, Eclipse's support is better.  The Eclipse wizard has an (i) icon that you can click on and F1 help that can point to the help URL directly.  (The help docs were just added so that still needs to be hooked up).
